airbyte-integrations/connectors/source-smartsheets/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-slim
+FROM python:3.10-slim
```

Comment:
The current implementation was using an old version of the smartsheets SDK which wouldn't support python 3.10 or newer and causes some other dependency issues when trying to build on top of it, it still runs on 3.9 and 3.8 but seemed appropriate to update the image to fit the more recent versions of the smartsheets SDK, I wouldn't say it's strictly necessary though since it's mostly innocuous I can revert it if need be.

Reply:
Let's keep this at `python:3.9-slim` for now. The CDK is tested with python3.8 and python3.9.

Reply:
This is done :D